.github/workflows/frontend.yaml

```diff
@@ -3,8 +3,12 @@ name: Frontend linting
 on:
   push:
     branches: [ main ]
+    paths:
+      - "app/frontend/**"
   pull_request:
     branches: [ main ]
+    paths:
+      - "app/frontend/**"
```

Comment:
This change makes sense since we're literally cd'ing into app/frontend before running prettier, so it's a pretty well scoped workflow.